datacenterlight/views.py

```diff
@@ -99,108 +98,8 @@ def get(self, request, *args, **kwargs):
-            'subject': 'DatacenterLight Beta Access Request',
-            'from_address': '(datacenterlight) datacenterlight Support <support@datacenterlight.ch>',
-            'to': form.cleaned_data.get('email'),
-            'from': '(datacenterlight) DatacenterLight Support support@datacenterlight.ch',
-            'context': context,
-            'template_name': 'request_access_confirmation',
-            'template_path': 'datacenterlight/emails/'
-        }
-        email = BaseEmail(**email_data)
-        email.send()
-
-        context.update({
-            'email': form.cleaned_data.get('email')
-        })
-
-        email_data = {
-            'subject': 'DatacenterLight Beta Access Request',
-            'from_address': '(datacenterlight) datacenterlight Support <support@datacenterlight.ch>',
-            'to': 'info@ungleich.ch',
-            'context': context,
-            'template_name': 'request_access_notification',
-            'template_path': 'datacenterlight/emails/'
-        }
-        email = BaseEmail(**email_data)
-        email.send()
-
-        messages.add_message(self.request, messages.SUCCESS,
-                             self.success_message)
-        return render(self.request, 'datacenterlight/beta_success.html', {})
-
-
-class BetaProgramView(CreateView):
-    template_name = "datacenterlight/beta.html"
-    model = BetaAccessVM
-    fields = '__all__'
-    # form_class = BetaAccessForm
-    # success_url = "/datacenterlight#requestform"
-    success_message = "Thank you, we will contact you as soon as possible"
-
-    def get_success_url(self):
-        success_url = reverse('datacenterlight:beta')
-        success_url += "#success"
-        return success_url
-
-    def get_context_data(self, **kwargs):
-        vms = BetaAccessVMType.objects.all()
-        context = super(BetaProgramView, self).get_context_data(**kwargs)
-
-        # templates = OpenNebulaManager().get_templates()
-        # data = VirtualMachineTemplateSerializer(templates, many=True).data
-
-        context.update({
-            'base_url': "{0}://{1}".format(self.request.scheme,
-                                           self.request.get_host()),
-            'vms': vms
-        })
-        return context
-
-    def post(self, request, *args, **kwargs):
-        data = request.POST
-        vms = BetaAccessVM.create(data)
-
-        context = {
-            'base_url': "{0}://{1}".format(self.request.scheme,
-                                           self.request.get_host()),
-            'email': data.get('email'),
-            'name': data.get('name'),
-            'vms': vms
-        }
-
-        email_data = {
-            'subject': 'DatacenterLight Beta Access Request',
-            'from_address': '(datacenterlight) datacenterlight Support <support@datacenterlight.ch>',
-            'to': 'info@ungleich.ch',
-            'context': context,
-            'template_name': 'request_beta_access_notification',
```

Comment:
related email template file `datacenterlight/templates/emails/request_beta_access_notification.html` can also be removed.